src/scythe-bidder/bidarea.tsx

```diff
@@ -158,7 +142,11 @@ const BidArea = (props: {
         dataIndex="mat"
         render={(mat: Mat) => {
           return (
-            <a href={matImages[mat]} target="_blank" rel="noopener noreferrer">
+            <a
+              href={process.env.PUBLIC_URL + MAT_IMAGES[mat]}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
```

Comment:
We should be able to import the assets here using webpack like `import InnovativeMatImg from './Innovative.png` here - not 100% sure on how to use it yet though

Reply:
I looked through the docs and I don't think webpack is configured for this project.

Reply:
So there is a restriction in ModuleScopePlugin of create-react-app to ensure that relative imports from app's source directory (src) don't reach outside of ( [source](https://stackoverflow.com/questions/44114436/the-create-react-app-imports-restriction-outside-of-src-directory) ). This behaviour can be changed by using eject but I'm not comfortable with the overhead that comes with it.

I see two ways to move ahead:
1. Move the images from public/ to src/public or src/scythe-bidder/public and use webpack imports
2. Keep the code as is with this improvement: replace process.env.PUBLIC_URL + MAT_IMAGES[mat] with just MAT_IMAGES[mat] and add process.env.PUBLIC_URL to the definition in constants.tx

@lightyearsaway Let me know what you think.

Reply:
Had a chat with @lightyearsaway and updated the code to use (2) above.